README.md

```diff
@@ -2,34 +2,43 @@
 
 # Leonardo
 
-Leonardo serves as a way to launch compute within the Terra security boundary. It does so via multiple different cloud hardware virtualization mechanisms, currently leveraging only the Google Cloud Platform.
+`leonardo` serves as a way to launch compute within the Terra security boundary. 
+It does so via multiple different cloud hardware virtualization mechanisms, currently leveraging only the Google Cloud Platform.
```

Comment:
Not necessarily the scope of your PR, but this is outdated - we offer GCP and Azure cloud services.

Reply:
I updated this to reflect both GCP and Azure